doc/api/child_process.markdown

```diff
@@ -455,23 +180,31 @@ leaner than [`child_process.exec()`][]. It has the same options.
   * `gid` {Number} Sets the group identity of the process. (See setgid(2).)
 * Return: ChildProcess object
 
-This is a special case of the [`child_process.spawn()`][] functionality for
-spawning Node.js processes. In addition to having all the methods in a normal
-ChildProcess instance, the returned object has a communication channel built-in.
-See [`ChildProcess#send()`][] for details.
+The `child_process.fork()` method is a special case of
+[`child_process.spawn()`][] used specifically to spawn new Node.js processes.
+Like `child_process.spawn()`, a `ChildProcess` object is returned. Unlike
+`child_process.spawn()` however, the returned `ChildProcess` will have an
+additional communication channel built-in that allows messages to be passed
+back and forth between the parent and child. See [`ChildProcess#send()`][] for
+details.
 
-These child Node.js processes are still whole new instances of V8. Assume at
-least 30ms startup and 10mb memory for each new Node.js. That is, you cannot
-create many thousands of them.
+Spawned child Node.js processes include their own whole new instances of V8.
+These require at least 30 milliseconds to startup and consume at least 10mb
+of memory for each new process. Because of these requirements, spawning a large
+number of child Node.js processes is not recommended.
 
-The `execPath` property in the `options` object allows for a process to be
-created for the child rather than the current `node` executable. This should be
-done with care and by default will talk over the fd represented an
-environmental variable `NODE_CHANNEL_FD` on the child process. The input and
-output on this fd is expected to be line delimited JSON objects.
+By default, `child_process.fork()` will spawn new Node.js instances using the
+`process.execPath` of the parent process. The `execPath` property in the
+`options` object allows for an alternative exec path to be used. This should be
+done with care as there are certain caveats that need to be understood.
```

Comment:
@bnoordhuis are there caveats? The IPC channel interface is pretty well defined, is it not? And can even be implemented by non-node processes. Also, execPath can be set in spawn and exec, as well (not using a property, but by just explicitly specifying some specific node to run).